autoload/selector.vim

```diff
@@ -12,34 +12,18 @@
 " See the License for the specific language governing permissions and
 " limitations under the License.
 
-""
-" @section Introduction, intro
```

Comment:
Should still have some kind of intro section, shouldn't it? The maktaba file won't have one because it would conflict with maktaba's own intro.

What edits do these source file changes write into the generated docs?